ansible-ipi-install/roles/shared-labs-prep/tasks/main.yml

```diff
@@ -285,6 +285,35 @@
       with_items: "{{ disable_nics }}"
   delegate_to: localhost
 
+- name: Create Manifest folder
+  block:
+    - name: Set manifest directory path
+      set_fact:
+        manifest_path: "{{ playbook_dir }}/roles/installer/files/manifests/"
+      
+    - name: Clean up any existing manifest content
+      file:
+        path: "{{ manifest_path }}"
+        state: absent
+
+    - name: Create manifest directory
```

Comment:
Why do you need to create this, looks like the folder already exists right?

Reply:
Doing this for the same reason to keep the folder updated for the current execution only.

Reply:
OK, I will take a look at the installer role again and review